### DISCUSSION TITLE
Problems with 3.6.2 Version

### DISCUSSION THREAD

**Discussion by piotrek1668** [amerkoleci/Vortice.Windows#458] [General]
After updating from Vortice 3.5.0 to 3.6.2 I get several issues. I saw, that there are some breaking changes like switching from `int` to `uint`, I have adjsuted my code but there are still some problems. Here some examples:

`System.MissingMethodException: 'Method not found: 'SharpGen.Runtime.Result Vortice.DXGI.IDXGIFactory6.EnumAdapterByGpuPreference(UInt32, Vortice.DXGI.GpuPreference, !!0 ByRef)'.'
`

`System.MissingFieldException: 'Field not found: 'Vortice.DXGI.AdapterDescription1.DeviceId'.'`

`System.MissingFieldException: 'Field not found: 'Vortice.DXGI.SwapChainDescription1.Width'.'`

Any idea what is going wrong here?

**Comment by amerkoleci** [comment]
Hi,
Probably you have mixed version of Vortice, did you updated all to 3.6.2 and recompiled the application?

**Reply by piotrek1668** [reply]
Yes, my renovate bot always updates all versions:

Package | Type | Update | Change
-- | -- | -- | --
Vortice.D3DCompiler | nuget | minor | 3.5.0 -> 3.6.2
Vortice.Direct2D1 | nuget | minor | 3.5.0 -> 3.6.2
Vortice.Direct3D11 | nuget | minor | 3.5.0 -> 3.6.2
Vortice.DirectInput | nuget | minor | 3.5.0 -> 3.6.2
Vortice.DirectX | nuget | minor | 3.5.0 -> 3.6.2
Vortice.XAudio2 | nuget | minor | 3.5.0 -> 3.6.2

I made a `git clean -xdf` on my repo and rebuild already. I will check again my Dlls

**Reply by piotrek1668** [reply]
ok, my fault. The bin output folder have the correct version of vortice-libs but not my application DLLs, they were still 3.5.0.

Thank you for your hint. So this issue is solved.
